scripts/install_python_dev_dependencies.py

```diff
@@ -87,7 +87,8 @@ def install_installation_tools() -> None:
 def install_dev_dependencies() -> None:
     """Install dev dependencies from COMPILED_REQUIREMENTS_DEV_FILE_PATH."""
     subprocess.run(
-        ['pip-sync', COMPILED_REQUIREMENTS_DEV_FILE_PATH],
+        ['pip', 'install', '--require-hashes', '--no-deps', '-r',
+        COMPILED_REQUIREMENTS_DEV_FILE_PATH],
```

Comment:
Can we keep `pip-sync` here? `pip install` and `pip-sync` are slightly different in that `pip-sync` will also uninstall any installed packages that are not in the requirements file, which ensures that the dev environment is always exactly what's reflected in the requirements file.

Any arg that can be passed to `pip install` can be passed to `pip-sync` like this:

`pip-sync requirements.txt --pip-args "--no-cache-dir --no-deps"`

So I think you can do something like this:

`['pip-sync', COMPILED_REQUIREMENTS_DEV_FILE_PATH, '--pip-args', '--require-hashes --no-deps'],`

Reply:
done